netbox/extras/management/commands/rqworker.py

```diff
@@ -14,6 +14,8 @@ class Command(_Command):
     of only the 'default' queue).
     """
     def handle(self, *args, **options):
+        # Run the worker with scheduler functionality
+        options['with_scheduler'] = True
```

Comment:
We might want to do something with the `--with-scheduler` command argument, since it will no longer have any effect. Not sure if there's an easy way to do that though. I doubt it's worth any significant effort.